packages/lib/src/core/types.ts

```diff
@@ -120,6 +120,7 @@ export interface CoreOptions {
 
     onOrderRequest?(resolve: PromiseResolve, reject: PromiseReject, data: PaymentData): Promise<void>;
 
+    onOrderCancel?(order: Order): void;
```

Comment:
Are we sure we return the `order` directly? I think we return an object which contains the `order` (by checking [this line](https://github.com/Adyen/adyen-web/blob/master/packages/lib/src/components/Dropin/components/DropinComponent.tsx#L103))